openshift/templates/deploy.dc.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ${NAME}
+  annotations:
+    openshift.io/display-name: "wps-api"
+    description: "Wildfire Predictive Services - APIent"
+    openshift.io/long-description: "Wildfire Predictive Services - Web Client"
+    tags: "wps,wps-api"
+    iconClass: icon-js
+    openshift.io/provider-display-name: "Government of British Columbia"
+    openshift.io/documentation-url: "https://github.com/bcgov/wps-api"
+    openshift.io/support-url: "https://github.com/bcgov/wps-api"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}-${SUFFIX}
+parameters:
+  - name: NAME
+    description: Module name
+    value: wps-api
+  - name: SUFFIX
+    description: Deployment suffix, e.g. pr-###
+    required: true
+  - name: DOCKER_REGISTRY
+    value: docker-registry.default.svc:5000
+  - name: PROJECT_TOOLS
+    value: auzhsi-tools
+  - name: CPU_REQUEST
+    description: Requested CPU
+    value: 1000m
+  - name: CPU_LIMIT
+    description: CPU upper limit
+    value: 1000m
+  - name: MEMORY_REQUEST
+    description: Requested memory
+    value: 2Gi
+  - name: MEMORY_LIMIT
+    description: Memory upper limit
+    value: 2Gi
+objects:
+  - apiVersion: v1
+    kind: ConfigMap
+    data:
+      env.origins: |
+        '${NAME}-${SUFFIX}.pathfinder.gov.bc.ca'
```

Comment:
${NAME}-${SUFFIX} won't work because we are allowing origins for web! @Sybrand It was Derek who's been changing the config map!. Can you change this to 'wps-web-${SUFFIX}.pathfinder.gov.bc.ca'

Reply:
That's what GitHub's blame option is for.  Done!